packages/tasks/src/tasks/zero-shot-object-detection/about.md

```diff
@@ -0,0 +1,38 @@
+## About the task
+
+Zero-shot object detection is a computer vision task to detect objects and its class in images, without any prior training or knowledge of the classes.
+
+Zero shot object detection works by transferring knowledge learnt during training of one model, to classify novel classes that was not present in the training data.
+
+## Use Cases
+
+### Object Search
+
+Zero-shot Object Detection models are used in image search. Smartphones use Object Detection models to detect entities (such as specific places or objects) and allow the user to search for the entity on the Internet.
+
+### Object Counting
+
+Zero-shot Object Detection models are used to count instances of objects in a given image, this can include counting the objects in warehouses or stores, or counting the number of visitors in a store. They are also used to manage crowds at events to prevent disasters.
```

Comment:
```suggestion
Zero-shot object detection models are used to count instances of objects in a given image. This can include counting the objects in warehouses or stores or the number of visitors in a store. They are also used to manage crowds at events to prevent disasters.
```